pyproject.toml

```
@@ -1,5 +1,5 @@
 [tool.black]
-line-length = 110
+line-length = 99
```

Please undo this, unless you've got a great argument for what looks like another arbitrary value?

```suggestion
line-length = 110
```